projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -91,7 +91,7 @@ private void UpdateBirthRecordIdentifier()
         /// <para>// Getter:</para>
         /// <para>Console.WriteLine($"NCHS identifier: {ExampleBirthRecord.BirthRecordIdentifier}");</para>
         /// </example>
-        [Property("Birth Record Identifier", Property.Types.String, "Birth Certification", "Birth Record identifier.", true, IGURL.CertificateNumber, true, 4)]
+        [Property("Birth Record Identifier", Property.Types.String, "Birth Certification", "Birth Record identifier.", true, IGURL.CertificateNumber, false, 4)]
         [FHIRPath("Bundle", "identifier")]
         public string BirthRecordIdentifier
         {
```

Comment:
The StateLocalIdentifier1 (below but I can only add comments here), also needs to point to an extension url in vital record. That url is not currently in the VitalRecord URL file and needs to be generated.